nebu/cli/main.py

```diff
@@ -281,7 +281,10 @@ def publish(content_dir, publication_message):
         sys.exit(1)
 
 
-_comfirmation_prompt = 'This could overwrite your config... continue?'
+_comfirmation_prompt = (
```

Comment:
Should be `_confirmation_prompt`, spelling needs to be fixed both here and in the decorator at [L324](https://github.com/Connexions/nebuchadnezzar/blob/7cafb2c7131db63d4e291f56cecaec07a84a9635/nebu/cli/main.py#L324).